### ISSUE TITLE
Font weights are broken again

### ISSUE THREAD

**Issue by aleksejsolovev** [MonoLisaFont/feedback#128]
* Font edition (i.e. trial, personal, plus, pro): plus
* Font version (i.e. 1.805): 1.806
* The type of the font file you are using (i.e. otf, ttf): ttf
* Operating system (i.e. MacOS, Windows): Linux
* Display resolution (i.e. 3072x1920): 1920x1080

Font version 1.806 weights are broken again exactly in the same way as in version 1.804, see issue #112. Only normal font has proper weight, light, medium and bold are broken.

`fc-scan` report for 1.806:
```
$ fc-scan ttf | grep weight
	weight: 0(i)(s)
	weight: 0(i)(s)
	weight: 0(i)(s)
	weight: 0(i)(s)
	weight: 0(i)(s)
	weight: 0(i)(s)
	weight: 80(i)(s)
	weight: 80(i)(s)
```

fontforge font info dialog for 1.805 and 1.806:
![ff-1 805](https://user-images.githubusercontent.com/96003044/155127866-83daddce-1967-4d82-8ca3-e84bd3c0b13a.png)
![ff-1 806](https://user-images.githubusercontent.com/96003044/155127887-1bb5e2c0-6a68-408b-b778-52ed70d73da6.png)

Screenshot in kitty terminal with 1.805:
![1 805](https://user-images.githubusercontent.com/96003044/155128134-0cf3650c-6724-4454-ba86-246d9b6ff73a.png)
Same with 1.806:
![1 806](https://user-images.githubusercontent.com/96003044/155128149-029a2d25-b4a0-4267-97d4-94c907552dd0.png)

It's fourth time this issue has been repeated, see #89, #110, #112 and now. Looks like every release after version 1.800 leads to the issue. May be something is wrong with release process? IMHO, adding this case to some automatic tests before release would be helpful.

**Comment by okonet** [comment]
@aleksejsolovev I'm very sorry for this! I agree that we should invest more into the automated release management process that prevents this and we will with a higher priority.

**Comment by MarcusSterz** [comment]
@okonet Can you confirm? I do not see any problem here.
@vikky49 Do you see this when you preview?
<img width="1089" alt="Bildschirmfoto 2022-02-22 um 16 11 35" src="https://user-images.githubusercontent.com/51958720/155164655-069b93be-9201-4db7-988d-5ab994afcd63.png">
<img width="1032" alt="Bildschirmfoto 2022-02-22 um 16 11 23" src="https://user-images.githubusercontent.com/51958720/155164671-6e82fa3d-11dc-43d8-a109-1272393e9014.png">
<img width="1026" alt="Bildschirmfoto 2022-02-22 um 16 10 19" src="https://user-images.githubusercontent.com/51958720/155164684-9067d122-2c48-4679-9403-466b4e684939.png">
<img width="1044" alt="Bildschirmfoto 2022-02-22 um 16 10 05" src="https://user-images.githubusercontent.com/51958720/155164693-599597ce-8318-41ea-aefb-6a97d6147091.png">

**Comment by vikky49** [comment]
@MarcusSterz  Actually my bad. It looks good not sure why i thought it was not working. Just reverified it and looks good

**Comment by bebraw** [comment]
Fixed in 1.807.

As a sidenote, my understanding is that it can be preferable to use OTF under Linux though TTF is an option. TTF tends to be better on Windows, though.
